### PR TITLE
Support previewing with LWC Dev Server and LWR Server

### PR DESCRIPTION
**Pull request by maliroteh-sf**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -4,4 +4,24 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-export default {};
+
```
Comment (maliroteh-sf) [inline src/index.ts:7]: re-exporting everything b/c almost all of them are used in our other projects.

File: src/common/AndroidLauncher.ts [hunk 2]
```diff
@@ -40,14 +40,16 @@ export class AndroidLauncher {
...
      */
     public async launchPreview(
         compName: string,
         projectDir: string,
         appBundlePath: string | undefined,
         targetApp: string,
         appConfig: AndroidAppPreviewConfig | undefined,
-        serverPort: string
+        serverPort: string,
+        targettingLwrServer: boolean = false
```
Comment (khawkins) [inline src/common/AndroidLauncher.ts:52]: Nit: this should be `targetingLwrServer`.

**Review by khawkins — APPROVED** [review]
Other than the nit variable name update in All the Places, this looks fine to me. Thanks for finally exporting all our stuff! 😄

File: src/common/AndroidLauncher.ts (hunk 2 above)
Comment (maliroteh-sf) [reply src/common/AndroidLauncher.ts:52]: oops

**Pull request merged**